bundles/org.corpus_tools.hexatomic.corpusedit/src/main/java/org/eclipse/wb/swt/SwtResourceManager.java

```diff
@@ -0,0 +1,454 @@
+/*******************************************************************************
+ * Copyright (c) 2011 Google, Inc. All rights reserved. This program and the accompanying materials
```

Comment:
```suggestion
 * Copyright (c) 2011 Google, Inc. 
 * All rights reserved. This program and the accompanying materials
```

Improve readability while keeping the style valid.